FiveCalls/FiveCalls/LocationHeader.swift

```diff
@@ -44,14 +49,16 @@ struct LocationHeader: View {
                     .font(.system(.title3))
                     .fontWeight(.medium)
             }
-            .padding(.leading)
+            .padding(.horizontal, usingRegularFonts() ? 15 : 5)
             .padding(.vertical, 10)
-            Image(systemName: "location.circle")
-                .imageScale(.large)
-                .symbolRenderingMode(.hierarchical)
-                .font(.title3)
-                .padding(.trailing)
-                .padding(.leading, 7)
+            if usingRegularFonts() {
```

Comment:
is the approach here to remove some iconography when dynamic type is set to a larger size? can you post some screenshots of how that changes the layout?

Reply:
In this case, the interface gets crowded with the largest fonts. Some of the text then gets truncated (which we could probably fix separately, if you want). Removing the location graphic helps to keep everything up higher / crowds out less of the issues list and ensures all the text is visible. Screen shots (12 pro, regular view w/ default size text, same with largest text, graphic removed for largest text):

Now I'm seeing that the 5 calls icon is still very small, which isn't ideal. But also, it doesn't add to the functionality.

![regular](https://github.com/5calls/ios/assets/1638675/7ef6f268-dd39-4a1a-9605-fa9f6ebfd05c)

![largeNoChange](https://github.com/5calls/ios/assets/1638675/c16ff170-8145-4375-b7fd-5ab2032ae1a0)

![largeImageRemoved](https://github.com/5calls/ios/assets/1638675/f796d2db-d5c6-42b9-acf2-0617ca689e10)

Reply:
You may also notice that for the largest fonts, the "What's important to you?" heading goes away. Screenshots (12 pro, very large text with header vs no header):
![header](https://github.com/5calls/ios/assets/1638675/312cc6ac-fe89-4dfc-82ad-372a9acc25f4)

![noHeader](https://github.com/5calls/ios/assets/1638675/377ee9f0-128f-4ec1-9c9a-4977fa0494fb)

Reply:
👍 this looks good, I usually see screens on social with a slightly larger text size, but not HUGE text like this so this seems like a decent tradeoff for the few people who need this giant size.